packages/cli-e2e/__tests__/react.specs.ts

```diff
@@ -82,14 +82,7 @@ describe('ui:create:react', () => {
       projectName
     );
 
-    await Promise.race([
-      buildTerminal.when('exit').on('process').do().once(),
-      buildTerminal
-        .when(/Happy hacking!/)
-        .on('stdout')
-        .do()
-        .once(),
-    ]);
+    await buildTerminal.when('exit').on('process').do().once();
```

Comment:
🤨 Why do we not race the Happy hacking anymore?

Reply:
because we need to wait for the server to finish its setup.